default/scripting/ship_hulls/ship_hulls.macros

```diff
@@ -204,11 +204,11 @@ HULL_FUEL_EFFICIENCY_EFFECTSGROUP
             scope = And [
                 Ship
                 Source
```

Comment:
Put the `Source` condition first, or just leave out the `Ship` condition entirely.